src/facade/facade.cc

```diff
@@ -39,6 +39,21 @@ ConnectionStats& ConnectionStats::operator+=(const ConnectionStats& o) {
   return *this;
 }
 
+ReplyStats& ReplyStats::operator+=(const ReplyStats& o) {
+  ADD(io_write_cnt);
```

Comment:
consider adding `static_assert(sizeof(ReplyStats) == X);` like we do in other `operator+=`
(I know it's just moved code, but why not)